src/kernel/src/pm/mod.rs

```diff
@@ -28,11 +28,21 @@ use crate::{
         ThreadManager,
     },
 };
+use ::core::sync::atomic::Ordering;
 use ::sys::{
     error::Error,
     pm::ProcessIdentifier,
 };
 
+//==================================================================================================
+// Constants
+//==================================================================================================
+
+// Use relaxed ordering for all atomic operations to mitigate synchronization overhead. It is safe
+// to use this ordering semantics because Nanvix is a single-core system, and the kernel runs with
+// interrupts disabled.
```

Comment:
Is it true that the kernel runs with interrupts disabled?

Reply:
Yes, it is.